src/Marten/StoreOptions.cs

```diff
@@ -219,9 +219,22 @@ public void Serializer(ISerializer serializer)
         /// <param name="enumStyle"></param>
         /// <param name="casing">Casing style to be used in serialization</param>
         /// <param name="collectionStorage">Allow to set collection storage as raw arrays (without explicit types)</param>
-        public void UseDefaultSerialization(EnumStorage enumStyle = EnumStorage.AsInteger, Casing casing = Casing.Default, CollectionStorage collectionStorage = CollectionStorage.Default)
+        /// <param name="nonPublicMembersStorage">Allow to use non public members should be used during deserialization</param>
```

Comment:
Change to "Allow non public members to be used during deserialization"

Reply:
Fixed 👍